pyproject.toml

```diff
@@ -111,13 +120,13 @@ exclude = [
 packages = ["src/icalendar"]
 
 [tool.hatch.metadata.hooks.vcs.urls]
-# This is a dynamic generation of [project.urls]
-Homepage = "https://icalendar.readthedocs.io/"
-Repository = "https://github.com/collective/icalendar/"
+# Dynamic URLs for built packages
+Homepage = "https://icalendar.readthedocs.io/en/stable/"
+Repository = "https://github.com/collective/icalendar"
 source_archive = "https://github.com/collective/icalendar/archive/{commit_hash}.zip"
 Issues = "https://github.com/collective/icalendar/issues"
-Documentation = "https://icalendar.readthedocs.io/"
-Changelog = "https://icalendar.readthedocs.io/en/latest/changelog.html"
+Documentation = "https://icalendar.readthedocs.io/en/stable/"
+Changelog = "https://icalendar.readthedocs.io/en/stable/changelog.html"
```

Comment:
Can you remove the duplicate entries? Everything that is not dynamically generated should be in the static part.